### PR TITLE
View sheds now have scaling bias

### PR DESCRIPTION
**Pull request by peterAdams1014**

### CODE REVIEW

**Comment by OsiJr** [comment]
No changes were required in Metal?

File: src/vcRender.cpp [hunk 1]
```diff
@@ -760,13 +761,14 @@ void vcRender_RenderAndApplyViewSheds(vcState *pProgramState, vcRenderContext *p
       // configure UD render to only render into portion of buffer
       vdkRenderView_SetTargetsWithPitch(pProgramState->pVDKContext, pRenderContext->viewShedRenderingContext.pRenderView, nullptr, 0, pRenderContext->viewShedRenderingContext.pDepthBuffer + (i * (ViewShedMapRes.x / ViewShedMapCount)), 0, ViewShedMapRes.x * 4);
       vdkRenderView_SetMatrix(pProgramState->pVDKContext, pRenderContext->viewShedRenderingContext.pRenderView, vdkRVM_Projection, shadowRenderCamera.matrices.projectionUD.a);
-
+
```
Comment (samsinsane) [inline src/vcRender.cpp]: How does this kind of thing keep happening? Are you using Notepad++ or something?

Comment (peterAdams1014) [reply src/vcRender.cpp]: EditorConfig has been reinstalled. This won't happen again.

**Comment by peterAdams1014** [comment]
> No changes were required in Metal?

Theres an open issue regarding view sheds not working in Metal, which will not be resolved this PR.

**Pull request merged**